src/storages/pluggable/SplitsCachePluggable.ts

```diff
@@ -0,0 +1,274 @@
+import { isFiniteNumber, toNumber, isNaNNumber } from '../../utils/lang';
+import KeyBuilder from '../KeyBuilder';
+import { ICustomStorageWrapper, ISplitsCacheAsync } from '../types';
+import { ILogger } from '../../logger/types';
+import { usesSegments } from '../AbstractSplitsCacheSync';
+import { ISplit } from '../../dtos/types';
+import { logPrefix } from './constants';
+import { SplitError } from '../../utils/lang/errors';
+
+/**
+ * ISplitsCacheAsync implementation for pluggable storages.
+ */
+export class SplitsCachePluggable implements ISplitsCacheAsync {
+
+  private readonly log: ILogger;
+  private readonly keys: KeyBuilder;
+  private readonly wrapper: ICustomStorageWrapper;
+
+  /**
+   *
+   * @param log logger instance
```

Comment:
As suggestiona:
1 - Constructor could omit being documented. Instead, you could document each variable prior to Constructor (Line 15-17)
2 - In order to make the documentation more clear: add 2 whitespaces between the param name and the description, and also treat the description as a sentence, starting with upper case and finishing with period.
example:  `* @param log logger instance`  ==> ```* @param log__Logger instance.``` (I added `_` since double whitespaces are not shown)

Reply:
Done 2